pyanaconda/modules/payloads/payload/rpm_ostree/rpm_ostree.py

```diff
@@ -188,8 +217,11 @@ def post_install_with_tasks(self):
 
         :return: list of tasks
         """
-        ostree_source = self._get_ostree_source()
+        # No extra steeps in case of the bootc install
+        if self._get_source(SourceType.BOOTC):
+            return []
 
+        ostree_source = self._get_ostree_source()
```

Comment:
Maybe we could check for BOOTC type of source rather here on ostree_source (_get_ostree_source is calling _get_source) ?